### DISCUSSION TITLE
Hello, it's Peadar from Tik Tok

### DISCUSSION THREAD

**Discussion by pea3nut** [joulev/ezkomment#119] [General]
Maybe we can communicate via Github? You can just at me on a specific line of code.

**Comment by joulev** [comment]
Hi Peadar! Okay I think communication on GitHub is fine too! I will wait for my teammate @VietAnh1010 to voice his opinions.

**Comment by VietAnh1010** [comment]
Hi Peadar! Communicating through GitHub is fine with me as well.

**Comment by joulev** [comment]
Okay. I have created a Discussion channel, [General](https://github.com/joulev/ezkomment/discussions/categories/general), where we can continue any conversations. The existing channel is renamed to [Questions](https://github.com/joulev/ezkomment/discussions/categories/questions) for Q/A.
